test/validate/windows_validate.go

```diff
@@ -100,6 +112,10 @@ func hnsStateFileIps(result []byte) (map[string]string, error) {
 	for _, v := range hnsResult {
 		if !v.IsRemoteEndpoint {
 			hnsPodIps[v.IPAddress.String()] = v.MacAddress
+			if v.IPv6Address.String() != "<nil>" {
+				hnsPodIps[v.IPv6Address.String()] = v.MacAddress
```

Comment:
Is this for dualstack windows ? Or is it some error we found ?

Reply:
This is for dualstack windows.